webapp/src/mutator.ts

```diff
@@ -322,6 +322,49 @@ class Mutator {
         )
     }
 
+    // Board Members
+
+    async createBoardMember(boardId: string, userId: string, description = 'create board member'): Promise<void> {
+        const member = { boardId, userId, schemeEditor: true } as BoardMember
+
+        await undoManager.perform(
+            async () => {
+                await octoClient.createBoardMember(member)
+            },
+            async () => {
+                await octoClient.deleteBoardMember(member)
+            },
+            description,
+            this.undoGroupId,
+        )
+    }
+
+    async updateBoardMember(newMember: BoardMember, oldMember: BoardMember, description = 'update board member'): Promise<void> {
+        await undoManager.perform(
+            async () => {
+                await octoClient.updateBoardMember(newMember)
+            },
+            async () => {
+                await octoClient.updateBoardMember(oldMember)
+            },
+            description,
+            this.undoGroupId,
+        )
+    }
+
+    async deleteBoardMember(member: BoardMember, description = 'delete board member'): Promise<void> {
```

Comment:
@chenilim Adding and removing members is something that we want to track as part of the undo/redo?